setup.py

```diff
@@ -54,7 +54,7 @@
             'coveralls~=3.0.0',  # depends on coverage as well
         ],
         'rest': [
-            'djangorestframework>=3.11,<3.12',
+            'djangorestframework~=3.12.0',
```

Comment:
Important change, can you please include in changelog? :smile:

Reply:
Done